test/src/main/java/org/apache/accumulo/test/AdminCheckIT.java

```diff
@@ -264,6 +272,113 @@ public void testAdminCheckRunWithCheckFailures() {
     assertTrue(out4.contains(expStatusInfo3And4));
   }
 
+  /*
+   * The following tests test the expected functionality of the admin check command (e.g., are the
+   * checks correctly identifying problems)
+   */
+
+  @Test
+  public void testSystemConfigCheck() throws Exception {
+    String table = getUniqueNames(1)[0];
+
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProperties()).build()) {
+      client.tableOperations().create(table);
+
+      ReadWriteIT.ingest(client, 10, 10, 10, 0, table);
+      client.tableOperations().flush(table, null, null, true);
+
+      // the slow compaction is to ensure we hold a table lock when the check runs, so we have
+      // locks to check
+      IteratorSetting is = new IteratorSetting(1, SlowIterator.class);
+      is.addOption("sleepTime", "10000");
+      CompactionConfig slowCompaction = new CompactionConfig();
+      slowCompaction.setWait(false);
+      slowCompaction.setIterators(List.of(is));
+      client.tableOperations().compact(table, slowCompaction);
+
+      var p = getCluster().exec(Admin.class, "check", "run", "system_config");
+      assertEquals(0, p.getProcess().waitFor());
+      String out = p.readStdOut();
+      assertTrue(out.contains("locks are valid"));
+      assertTrue(out.contains("Check SYSTEM_CONFIG completed with status OK"));
+    }
+  }
+
+  @Test
+  public void testPassingMetadataTableCheck() throws Exception {
+    // Tests the METADATA_TABLE check in the case where all checks pass
+
+    // no extra setup needed, just check the metadata table
+    var p = getCluster().exec(Admin.class, "check", "run", "metadata_table");
+    assertEquals(0, p.getProcess().waitFor());
+    String out = p.readStdOut();
+    assertTrue(out.contains("Looking for offline tablets"));
+    assertTrue(out.contains("Checking some references"));
+    assertTrue(out.contains("Looking for missing columns"));
+    assertTrue(out.contains("Looking for invalid columns"));
+    assertTrue(out.contains("Check METADATA_TABLE completed with status OK"));
```

Comment:
Could look for other checks running in this test and others that specify a check.

```suggestion
    assertTrue(out.contains("Check METADATA_TABLE completed with status OK"));
    // Should not see other checks running
    assertFalse(out.contains("SYSTEM_CONFIG"));
```